packages/frontend/app/pods/components/pod/chat/chat-entry/emoji-parser.ts

```diff
@@ -0,0 +1,23 @@
+const emojiDict = new Map([
+    [":partyparrot:", "'(\\(0>0)/)'"] // TODO: use gif
+]);
+
+export default class EmojiParser {
+
+    parseEmoji(message: string): string {
+        let emojiRegex = /:.*:/gi;
+        let matches = message.match(emojiRegex);
+        let emojifiedMessage : string = message;
+    
+        if (matches !== null) {
```

Comment:
this whole block could maybe be simplified a bit by using the function replacer variant of the `replace` method?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Using_an_inline_function_that_modifies_the_matched_characters